youtube_dl_gui/locale/zh_CN/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -0,0 +1,582 @@
+"Last-Translator: Sofronio <xbox2xbox@gmail.com>\n"
+"Language-Team: en\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: pygettext.py 1.5\n"
+
+#: formats.py:9 formats.py:108
+msgid "ID"
+msgstr "用户名"
+
+#: formats.py:10 formats.py:109 mainframe.py:140
+msgid "Title"
+msgstr "标题"
+
+#: formats.py:11 formats.py:110
+msgid "Title + ID"
+msgstr "标题 + 用户名"
+
+#: formats.py:12 formats.py:111
+msgid "Title + Quality"
+msgstr "标题 + 视频质量"
+
+#: formats.py:13 formats.py:112
+msgid "Title + ID + Quality"
+msgstr "标题 + 用户名 + 视频质量"
+
+#: formats.py:14 formats.py:113
+msgid "Custom"
+msgstr "自定义"
+
+#: formats.py:19 formats.py:118 mainframe.py:498 mainframe.py:501
+msgid "default"
+msgstr "默认"
+
+#: mainframe.py:97
+msgid "Enter URLs below"
+msgstr "在下方输入地址"
+
+#: mainframe.py:98
+msgid "Update"
+msgstr "更新"
+
+#: mainframe.py:99 optionsframe.py:41
+msgid "Options"
+msgstr "选项"
+
+#: mainframe.py:100 optionsframe.py:587
+msgid "Stop"
+msgstr "停止"
+
+#: mainframe.py:101
+msgid "Info"
+msgstr "信息"
+
+#: mainframe.py:102
+msgid "Welcome"
+msgstr "欢迎"
+
+#: mainframe.py:103
+msgid "Warning"
+msgstr "警告"
+
+#: mainframe.py:105
+msgid "Add"
+msgstr "添加"
+
+#: mainframe.py:106
+msgid "Download list"
+msgstr "下载队列"
+
+#: mainframe.py:107 mainframe.py:511 mainframe.py:529
+msgid "Delete"
+msgstr "删除"
+
+#: mainframe.py:108
+msgid "Play"
+msgstr "播放"
+
+#: mainframe.py:109
+msgid "Up"
+msgstr "提前"
+
+#: mainframe.py:110
+msgid "Down"
+msgstr "下降"
+
+#: mainframe.py:111
+msgid "Reload"
+msgstr "重新加载"
+
+#: mainframe.py:112 mainframe.py:443 mainframe.py:644
+msgid "Pause"
+msgstr "暂停"
+
+#: mainframe.py:113 mainframe.py:860 mainframe.py:861 optionsframe.py:585
+msgid "Start"
+msgstr "开始"
+
+#: mainframe.py:114
+msgid "About"
+msgstr "关于"
+
+#: mainframe.py:115
+msgid "View Log"
+msgstr "查看日志"
+
+#: mainframe.py:117
+msgid "Successfully downloaded {0} URL(s) in {1} day(s) {2} hour(s) {3} minute(s) {4} second(s)"
+msgstr "成功下载{0}个链接 用时{1}天{2}小时{3}分钟{4}秒"
+
+#: mainframe.py:119
+msgid "Downloads completed"
+msgstr "下载完成"
+
+#: mainframe.py:120
+msgid "Total Progress: {0:.1f}% | Queued ({1}) Paused ({2}) Active ({3}) Completed ({4}) Error ({5})"
+msgstr "总进度：{0:.1f}% | 队列 ({1}) 暂停 ({2}) 活动 ({3}) 完成 ({4}) 错误 ({5})"
+
+#: mainframe.py:121
+msgid "Stopping downloads"
+msgstr "正在停止下载"
+
+#: mainframe.py:122
+msgid "Downloads stopped"
+msgstr "下载已经停止"
+
+#: mainframe.py:123
+msgid "You need to provide at least one URL"
+msgstr "请至少提供一个链接"
+
+#: mainframe.py:124
+msgid "Downloads started"
+msgstr "下载已经开始"
+
+#: mainframe.py:125
+msgid "Choose Directory"
+msgstr "选择目录"
+
+#: mainframe.py:127
+msgid "Download in progress. Please wait for all downloads to complete"
+msgstr "正在下载中 请等待所有下载完毕"
+
+#: mainframe.py:128
+msgid "Update already in progress"
+msgstr "更新进行中"
+
+#: mainframe.py:130
+msgid "Downloading latest youtube-dl. Please wait..."
+msgstr "正在下载最新版本的youtube-dl 请稍候"
```

Comment:
If i missed something you can go ahead and fix it, but you should only edit the **msgstr** fields and *NOT* the **msgid**.